main.go

```diff
@@ -38,7 +144,175 @@ func httpHandler(w http.ResponseWriter, r *http.Request) {
 			" Please check back again later.")); err != nil {
 			log.Printf("error during Write: %v", err)
 		}
+	case http.MethodPost:
+		http.HandleFunc("/verifyProject", verifySignature)
 	default:
 		http.Error(w, "only GET method is allowed", http.StatusMethodNotAllowed)
 	}
 }
+
+func verifyScorecardWorkflow(workflowContent string) bool {
```

Comment:
make this function return `error`, this way you can return different errors depending on what check fails. It'll help with troubleshooting

Reply:
+1